ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/participant/util/ApiToLfEngineSpec.scala

```diff
@@ -151,13 +151,13 @@ class ApiToLfEngineSpec extends WordSpec with Matchers {
     }
 
     "handle Decimals exceeding scale correctly" in {
```

Comment:
now we aren't testing `ApiToLfEngine` anymore -- wrap the decimal in a value and test them through `ApiToLfEngine`